src/template.js

```diff
@@ -123,15 +150,17 @@ switch (data.method) {
       break;
     }
 
-    const viewedObjectIDsOptions = {
+    const payload = {
+      eventType: 'view',
       eventName: data.eventName,
       index: data.index,
       objectIDs: formatValueToList(data.objectIDs),
       userToken: data.userToken,
     };
+    const chunks = chunkPayload(payload, ['objectIDs'], MAX_OBJECT_IDS);
```

Comment:
I'm not sure we want to do that work ourselves without explicit buy-in from the customer. My concern is that this would generate duplicate clicks and conversions, which could lead to some surprising effects in some features (after thinking carefully, I think most features would be fine because they deduplicate by usertoken/queryID pairs but I am not 100% sure). 

This is probably an edge case, but I would let the customer decide what the behaviour should be in that situation, as we're basically trying to circumvent the API limitation in one way when other solutions (such dropping all the objectIDs after the first 20) are eligible.

Reply:
Huh, I naively thought 'those payloads have different objectIDs or filters. so it's fine.'

Reply:
Actually this is right before HTTP request happens. So we need to do something here, otherwise the request will fail. We can provide an option (dropping extra vs multiple events) upfront. Or we could add extra explanation to the documentation, so that they can pass proper number of objectIDs to the method in the first place.

However, considering there are lots of non tech savvy gtm users, it might increase complexity (already the doc and the integration is quite lengthy.) Plus, my another concern is, we proactively give control to users, we won't take it back when we decide to increase the limit, or even remove it.

Reply:
> Plus, my another concern is, we proactively give control to users, we won't take it back when we decide to increase the limit, or even remove it.

Hmm, that's a very good point! I'll dig into what could go wrong with duplicate events and come back to you if there's anything really problematic!

Reply:
Thank you for digging!